File: src/main/java/dev/slne/discord/exception/command/PreTicketCommandException.java
package dev.slne.discord.exception.command;

import java.io.Serial;

public class PreTicketCommandException extends PreCommandCheckException{

  @Serial
  private static final long serialVersionUID = -1353290082500326289L;

  public PreTicketCommandException(String message) {
    super(message);
  }

  public PreTicketCommandException(String message, Throwable cause) {
    super(message, cause);
  }
}

File: src/main/kotlin/dev/slne/discordold/exception/command/pre/PreTicketCommandException.kt
package dev.slne.discordold.exception.command.pre

import dev.slne.discordold.exception.DiscordException
import java.io.Serial

class PreTicketCommandException : PreCommandCheckException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(exception: DiscordException) : super(exception)

    companion object {
        @Serial
        private val serialVersionUID = -1353290082500326289L
    }
}
